extensions/positron-reticulate/src/extension.ts

```diff
@@ -123,13 +189,119 @@ export class ReticulateRuntimeManager implements positron.LanguageRuntimeManager
 		}
 	}
 
+	setSessions(hostRSessionId: string, reticulateId: string, session: positron.LanguageRuntimeSession) {
+		let sessionsMap: { reticulateSessionId: string; hostRSessionId: string; reticulateId: string }[] =
```

Comment:
This type is repeated many times -- could it be placed into a named interface for reuse?

Reply:
Done in https://github.com/posit-dev/positron/pull/7024/commits/6ea81395947f04cc8d309135fe49767136050ba0